.github/workflows/validation.yml

```
@@ -99,6 +99,32 @@ jobs:
           cd framework/pipeline/fileopening
           restManager PrintTrees v2.2.30_hits.root
 
+  macros-health:
+    name: "Macros with clean error output"
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/lobis/root-geant4-garfield:rest-for-physics
+    needs: [ framework-install ]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          repository: rest-for-physics/framework
+          path: framework
+      - name: Restore cache
+        uses: actions/cache@v3
+        id: framework-install-cache
+        with:
+          key: ${{ env.BRANCH_NAME }}-${{ github.sha }}
+          path: ${{ env.REST_PATH }}
+      - name: Load restRootMacros
+        run: |
+          source ${{ env.REST_PATH }}/thisREST.sh
+          cd framework/pipeline
+          export DISPLAY=localhost:0.0
+          restRoot -b -q 2> error.log
```

```suggestion
          restRoot -b -q -m 1 2> error.log
```
I think you should do ´restRoot -m 1´ to load the macros, in fact now macros are giving warnings/errors